src/io/flutter/inspector/FrameRenderingDisplay.java

```diff
@@ -37,41 +40,47 @@ public static JPanel createJPanelView(Disposable parentDisposable, FlutterApp ap
     assert app.getVMServiceManager() != null;
     final FlutterFramesMonitor flutterFramesMonitor = app.getVMServiceManager().getFlutterFramesMonitor();
 
-    final FPSPanel fpsPanel = new FPSPanel(flutterFramesMonitor);
-
-    final JPanel labelsPanel = new JPanel();
-    labelsPanel.setOpaque(false);
-    labelsPanel.setLayout(new BoxLayout(labelsPanel, BoxLayout.X_AXIS));
-    panel.add(fpsPanel);
-    panel.add(labelsPanel);
-
-    final JBLabel fpsLabel = new JBLabel();
-    fpsLabel.setAlignmentY(Component.TOP_ALIGNMENT);
-    fpsLabel.setFont(UIUtil.getLabelFont(UIUtil.FontSize.SMALL));
-    fpsLabel.setForeground(UIUtil.getLabelDisabledForeground());
-    fpsLabel.setOpaque(false);
-    fpsLabel.setBorder(JBUI.Borders.empty(4));
-
-    final JBLabel elapsedLabel = new JBLabel("", SwingConstants.RIGHT);
-    elapsedLabel.setAlignmentY(Component.TOP_ALIGNMENT);
-    elapsedLabel.setFont(UIUtil.getLabelFont(UIUtil.FontSize.SMALL));
-    elapsedLabel.setForeground(UIUtil.getLabelDisabledForeground());
-    elapsedLabel.setOpaque(false);
-    elapsedLabel.setBorder(JBUI.Borders.empty(4));
-
-    labelsPanel.add(fpsLabel);
-    labelsPanel.add(Box.createHorizontalGlue());
-    labelsPanel.add(elapsedLabel);
+    final FrameRenderingPanel frameRenderingPanel = new FrameRenderingPanel(flutterFramesMonitor);
+
+    final JBLabel latestFrameTimeLabel = new JBLabel();
+    latestFrameTimeLabel.setAlignmentX(Component.RIGHT_ALIGNMENT);
+    latestFrameTimeLabel.setFont(UIUtil.getLabelFont(UIUtil.FontSize.SMALL));
+    latestFrameTimeLabel.setForeground(UIUtil.getLabelDisabledForeground());
+    latestFrameTimeLabel.setBorder(JBUI.Borders.empty(0, 4));
```

Comment:
I did this because I thought the 16ms looked a little too far indented with 4. Plus the more indentation the label has, the further we would need to push the dotted line in.